Engines/Wine/Engine/Implementation/script.js

```diff
@@ -23,6 +23,7 @@ module.default = class WineEngine {
         this._ldPath = propertyReader.getProperty("application.environment.ld");
         this._wineEnginesDirectory = propertyReader.getProperty("application.user.engines") + "/wine";
         this._winePrefixesDirectory = propertyReader.getProperty("application.user.containers") + "/" + WINE_PREFIX_DIR + "/";
+        this._useRuntime = (propertyReader.getProperty("application.environment.wineRuntime") !== "false");
```

Comment:
Is it possible to set this property from inside Phoenicis?

Reply:
No. It would be a bit problematic to do that because Phoenicis should be agnostic of the Engine and this property is specific for Wine.

Reply:
Ok